Immense.RemoteControl.Server/Hubs/DesktopHub.cs

```diff
@@ -146,6 +146,12 @@ public override async Task OnDisconnectedAsync(Exception? exception)
         await base.OnDisconnectedAsync(exception);
     }
 
+    public async Task<string> PingViewer(string viewerConnectionId)
+    {
+        using var cts = new CancellationTokenSource(5_000);
```

Comment:
Minor, but I'm a big fan of supplying `TimeSpan` for anything to do with time. 
```suggestion
        using var cts = new CancellationTokenSource(TimeSpan.FromMilliseconds(5_000));
```

Reply:
Agreed.  Thanks!